packages/workspace-fusion/src/lib/types/fusionController.ts

```diff
@@ -38,3 +38,37 @@ export type FusionWorkspaceError = {
 	detail: string;
 	title: string;
 };
+
+/** API for manipulating the workspace */
+export type WorkspaceController<
+	TData extends Record<PropertyKey, unknown>,
+	TContext extends Record<PropertyKey, unknown> = never,
+	TCustomSidesheetEvents extends BaseEvent = never
+> = {
```

Comment:
```suggestion
export type WorkspaceController<
	TData extends Record<PropertyKey, unknown>,
	TCustomSidesheetEvents extends BaseEvent = never,
	TContext extends Record<PropertyKey, unknown> = never,
> = {
```

I think this order makes more sense if an App developer has to pass generics (i guess this is more relevant for SidesheetConfig type).